src/Components/Tabs.re

```diff
@@ -49,7 +49,7 @@ let schedulePostRender = f => postRenderQueue := [f, ...postRenderQueue^];
 let component = React.Expert.component("Tabs");
 let make =
     (
-      ~children as render: 'a => element,
+      ~children as render: (~isSelected: bool, int, 'a) => element,
```

Comment:
I wonder if this `int` should be a named parameter as well, like `~index:int`?

Reply:
Fixed in https://github.com/onivim/oni2/pull/2013/commits/f43e24dfdc0e5346d08bdf825c2c8143fa3979e3